api/v1beta2/foundationdbbackup_types.go

```diff
@@ -320,14 +323,32 @@ func (configuration *BlobStoreConfiguration) getURL(backup string, bucket string
 	if configuration.AccountName == "" {
 		return ""
 	}
-
-	var sb strings.Builder
+	var (
+		defaultPort string
+		sb          strings.Builder
+	)
+	if !strings.Contains(configuration.AccountName, ":") { // add default http port
```

Comment:
It's probably better to do something like this:

```go
backupURL := &url.URL{Host: "account@account:931"}
if backupURL.port() == "" {
    defaultPort = ":443"
}
```

If we use the current approach IPv6 addresses will be unhappy :)

In general it probably would be good to refactor this setup to make use of the net.Url package from go instead of putting a raw string together. Nothing that you have to fix in this PR, just something I noticed looking at the PR. I'll create an issue for that follow up -> https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1987

Reply:
good point! I wasn't sure how urls would handle the @ but I will dig at it

Reply:
it handled it fine